packages/fxa-settings/src/pages/ResetPassword/index.test.tsx

```diff
@@ -73,16 +86,50 @@ describe('PageResetPassword', () => {
     expect(screen.queryByLabelText('Email')).not.toBeInTheDocument();
   });
 
-  it('renders a "Remember your password?" link if "canGoBack: true"', () => {
-    render(<ResetPassword canGoBack={true} />);
+  it('emits a metrics event on render', () => {
+    render(<ResetPassword />);
+    expect(logPageViewEvent).toHaveBeenCalledWith(viewName, REACT_ENTRYPOINT);
+  });
 
-    expect(
-      screen.getByRole('link', { name: 'Remember your password? Sign in' })
-    ).toBeInTheDocument();
+  it('submit success', async () => {
+    const account = {
+      resetPassword: jest.fn().mockResolvedValue({
+        passwordForgotToken: '123',
+      }),
+    } as unknown as Account;
+
+    renderWithAccount(account);
+
+    await act(async () => {
+      fireEvent.input(screen.getByTestId('reset-password-input-field'), {
+        target: { value: MOCK_ACCOUNT.primaryEmail.email },
+      });
+    });
+
+    await act(async () => {
+      fireEvent.click(screen.getByText('Begin Reset'));
+    });
+
+    expect(account.resetPassword).toHaveBeenCalledWith(
+      MOCK_ACCOUNT.primaryEmail.email
+    );
+    expect(mockNavigate).toHaveBeenCalledWith('confirm_reset_password', {
+      replace: true,
+    });
   });
 
-  it('emits a metrics event on render', () => {
-    render(<ResetPassword />);
-    expect(usePageViewEvent).toHaveBeenCalledWith(viewName, REACT_ENTRYPOINT);
+  it('displays tooltip on errors - unknown account', async () => {
+    const gqlError: any = AuthUiErrorNos[102]; // Unknown account error
+    const account = {
+      resetPassword: jest.fn().mockRejectedValue(gqlError),
+    } as unknown as Account;
+
+    renderWithAccount(account);
+
+    await act(async () => {
+      fireEvent.click(screen.getByText('Begin Reset'));
+    });
+
+    screen.getByText('Sign up');
```

Comment:
I _think_ in your previous test we need `await act(async () => {` since you're checking function calls next but when you're checking for `getBy` stuff you can use `await findBy` instead:

```
fireEvent.click(screen.getByText('Begin Reset'));
await screen.findByText('Sign up');
```